content/basics/use-ably.textile

```diff
@@ -0,0 +1,119 @@
+---
+title: How to use Ably
+meta_description: "A brief introduction on the best options available for using Ably"
+---
+
+This topic will briefly cover some of the APIs, SDKs, and adapters that Ably provides.
+
+h2(#sdks). Ably SDKs with Realtime and REST interfaces
+
+Ably provides client library SDKs that provide both a realtime interface, allowing persistent connections with low latency publishing and subscribing, and REST interface, communicating using the HTTP protocol and is effectively stateless. Ably SDKs are consistent across every language offering pub/sub functionality, presence, token and basic authentication, symmetric encryption and access to channel, presence and metrics history.
+
+Ably provides "a number of SDKS":https://ably.com/download in the most popular languages.
```

Comment:
I think instead of "the most popular languages." we should put some examples like, JS etc. then put and others.